app/src/main/java/com/codepath/travel/activities/CreateStoryActivity.java

```diff
@@ -147,6 +171,38 @@ private void setUpRecyclerView() {
         mItemTouchHelper.attachToRecyclerView(rvStoryPlaces);
     }
 
+    private void addSelectedPlaceInTrip(Place place, String photoReference) {
+        StoryPlace storyPlace = new StoryPlace(mNewTrip, place);
+        storyPlace.setThumbnail(photoReference);
```

Comment:
I think you should just delete the coverUrl field then. Right now we only ever display a photo for the story place as a thumbnail anyway. I had changed the getCoverUrl to support the default link if there was no set value, so I think you can do the same with the thumbnail. You'll just have to update the `onSetStoryPlaceCoverPhoto` method in the StoryActivity